.travis.yml

```diff
@@ -9,7 +9,8 @@ os:
 
 script:
   - ./autogen.sh
-  # clang might warn about C11 generic selections in isnan()
-  - CFLAGS=-Wno-c11-extensions ./configure --enable-werror
+  - ./configure --enable-werror
   - make -k
-  - CFLAGS=-Wno-c11-extensions make distcheck DISTCHECK_CONFIGURE_FLAGS=--enable-werror
+  - make distcheck DISTCHECK_CONFIGURE_FLAGS=--enable-werror
+  - sudo make install
+  - make installcheck
```

Comment:
What does this change?

Reply:
`-Wno-c11-extensions` is added via the configure script.

`make install` and `make installcheck` ensures installing htop works.